bin/file_extension.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+
+from reusable import json_dump, package_validation_dict
+
+
+def file_extension_test(file_path: str):
+    """
+    1. Reads file extension from provided path
+    2. Checks for valid file extension (.txt is used for now, future version will use ome.tiff and ome.tif)
+    3. Returns result of File extension test ("pass" or "fail")
+
+    Args:
+        file_path (str): Path to the downloaded file
+
+    Returns:
+        str: Result of File extension test
+    """
+    ext = file_path.split(".", 1)[-1]
+    # Do we want the extension check to be hard-coded like this or check against the file type data provided in the CSV?
+    # We are not using that column of the CSV at the moment.
+    file_ext_test = (ext == "ome.tiff" or ext == "ome.tif") 
```

Comment:
Instead of doing the ext split 

```
"my.test.file.ext".split(".", 1)[-1]
> 'test.file.ext'
```

My suggestion is to use:

```
.endswith(("ome.tiff", "ome.tif"))
```

Reply:
That said, I think using the `.split(".", 1)` could be more generalizable.  We could probably push back on people having more than 2 periods in their file name?

Reply:
as long as the first period is the beginning of the extension this way should be fine, and I agree it is simple. I will leave it for now

Reply:
I also prefer the approach using `.endswith()`, especially since it conveniently accepts multiple suffixes. My rationale is that I often append "tags" in the file name that are delimited by periods. For example:

```
mutations.passed.grch37.filt.annot.vcf.gz
```